packages/icons/src/icon/index.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * WordPress dependencies
+ */
+import { cloneElement, forwardRef } from '@wordpress/element';
+
+/**
+ * External dependencies
+ */
+import type { ReactElement, ForwardedRef } from 'react';
+import type { SVGProps } from '@wordpress/primitives';
```

Comment:
Along with the other suggestions, we may not need the React import here
```suggestion
import type { SVGProps } from '@wordpress/primitives';
```